docs/buildah.md

```diff
@@ -66,7 +66,7 @@ specify additional options via the `--storage-opt` flag.
 
 **--storage-opt** **value**
 
-Storage driver option, Default storage driver options are configured in /etc/containers/storage.conf. The `STORAGE_OPTS` environment variable overrides the default.  The --storage-opt specified options overrides all.
+Storage driver option, Default storage driver options are configured in /etc/containers/storage.conf (`$HOME/.config/containers/storage.conf` in rootless mode). The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all.
```

Comment:
nit of a nit, and not in your changes, but "Default storage" should be "default storage", or we should replace the preceding comma with a period and change to "The default" like in --storage-driver.  Ignoring or doing in a follow up is fine.